heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -109,14 +108,14 @@ private static int runSyncProcess(
     final StringBuilder pStdErr = stderr == null ? new StringBuilder() : stderr;
 
     // Log the command for debugging
-    LOG.log(Level.FINE, "Process command: `$ {0}`", Arrays.toString(cmdline));
+    LOG.log(Level.INFO, "Process command: ``{0}''", String.join(" ", cmdline));
     ProcessBuilder pb = getProcessBuilder(isInheritIO, cmdline, workingDirectory, envs);
 
     Process process;
     try {
       process = pb.start();
     } catch (IOException e) {
-      LOG.log(Level.SEVERE, "Failed to run Sync Process ", e);
+      LOG.log(Level.SEVERE, "Failed to run sync process ", e);
```

Comment:
Yes the command. This is always good to log in the exception so you don't have to dig through previous logs to find all others commands run, and then guess which one failed.